tests/10-Bitstring.spec.js

```diff
@@ -12,16 +12,63 @@ describe('Bitstring', () => {
   });
 
   it('should fail to create an instance if no length', async () => {
+    let bitstring;
     let err;
     try {
-      new Bitstring();
+      bitstring = new Bitstring({length: undefined});
     } catch(e) {
       err = e;
     }
+    should.not.exist(bitstring);
     should.exist(err);
     err.name.should.equal('TypeError');
   });
 
+  it('should fail to create an instance if buffer and length', async () => {
+    let bitstring;
+    let err;
+    try {
+      bitstring = new Bitstring({length: 'foo', buffer: 'bar'});
+    } catch(e) {
+      err = e;
+    }
+    should.not.exist(bitstring);
+    should.exist(err);
+    err.should.be.instanceof(Error);
+    err.message.should.contain('one of "length" or "buffer"');
+  });
+
+  it('fails to create instance if length is not positive integer', async () => {
+    let bitstring;
+    let err;
+    try {
+      bitstring = new Bitstring({length: -2.2});
+    } catch(e) {
+      err = e;
+    }
+    should.not.exist(bitstring);
+    should.exist(err);
+    err.should.be.instanceof(TypeError);
+    err.message.should.contain('positive integer');
+  });
+
+  it('fails to create instance if position is not positive integer',
+    async () => {
+      let bitstring;
+      let err;
+      let position;
```

Comment:
```suggestion
```

Reply:
Position isn't used outside of the try/catch scope below *and* it is only ever set once, so we can remove this here and declare it a const internally below.